DNS/DNS_Cache.c

```diff
@@ -226,7 +231,7 @@
         int x;
 
         /* For each entry in the DNS cache table. */
-        for( x = 0; x < ipconfigDNS_CACHE_ENTRIES; x++ )
+        for( x = 0; x < ( int ) ipconfigDNS_CACHE_ENTRIES; x++ )
```

Comment:
Apologies for being a stickler for convention - but why not make the variable `x` unsigned rather than casting the value in test expression?

Reply:
I see the rationale: `index = x;`.
We'll need to cast at one place or another.